app/src/main/java/com/android/streetworkapp/model/moderation/PerspectiveAPIRepository.kt

```diff
@@ -22,7 +24,7 @@ class PerspectiveAPIRepository(private val client: OkHttpClient) : TextModeratio
    * @return True if the text is under all thresholds, false if the text is over at least one
```

Comment:
Suggestion: i would recommend using a companion object to store DEBUG_PREFIX as a constant (Line 18): 
```
companion object {
        private const val DEBUG_PREFIX = "PerspectiveAPIRepository:"
    }
```
(I cannot comment unchanged lines but it seemed relevant for this PR)

Reply:
done